packages/integration-objects/srv/utils/ErrorStatuses.js

```diff
@@ -0,0 +1,10 @@
+/* Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. */
+const ErrorStatuses = {
```

Comment:
@vidhyakpai Consider renaming this to `ErrorStatus` because this is semantically an enum. (But don't do it now, add it to the renaming ticket ;) )